src/main/java/org/gridsuite/directory/server/NotificationService.java

```diff
@@ -58,6 +59,14 @@ public void emitDirectoryChanged(UUID directoryUuid, String elementName, String
                 .setHeader(HEADER_ERROR, error);
         sendUpdateMessage(messageBuilder.build());
     }
+
+    public void emitDeletedStudy(UUID studyUuid, String userId) {
+        MessageBuilder<String> messageBuilder = MessageBuilder.withPayload("")
+                .setHeader(HEADER_USER_ID, userId)
+                .setHeader(HEADER_STUDY_UUID, studyUuid)
```

Comment:
Maybe we should use HEADER_ELEMENT_UUID here because directory-server manage elements not only studies

Reply:
since it's in the emitDeletedStudy it's ok to have study_uuid since it's only for studies (for now), if we need to change it later it won't be an issue.